templates/course_info.html

```diff
@@ -43,42 +43,70 @@ <h2 class="sub-header">Assistant(s) for this year ({{ course.year }} - {{ course
+                            </li>
+                        </ul>
 
-                <!-- Tab panes -->
-                <div class="tab-content">
-                    <!-- Assistants Pane -->
-                    <div class="tab-pane fade show active" id="assistants" role="tabpanel"
-                         aria-labelledby="assistants-tab">
-                        <table class="table table-hover mt-3">
-                            <thead>
-                            <tr>
-                                <th>Last name</th>
-                                <th>First name</th>
-                            </tr>
-                            </thead>
-                            <tbody>
-                            </tbody>
-                        </table>
-                    </div>
-                    <!-- Evaluations Pane -->
-                    <div class="tab-pane fade show" id="evaluations" role="tabpanel" aria-labelledby="evaluations-tab">
-                        <!-- Content for Evaluations -->
-                        <br>
-                        <p>Content for evaluations will go here.</p>
+                        <!-- Tab panes -->
+                        <div class="tab-content">
+                            <!-- Assistants Pane -->
+                            <div class="tab-pane fade show active" id="assistants-{{ course.year }}" role="tabpanel"
+                                 aria-labelledby="assistants-tab-{{ course.year }}">
+                                <table class="table table-hover mt-3">
+                                    <thead>
+                                    <tr>
+                                        <th>Last name</th>
+                                        <th>First name</th>
+                                    </tr>
+                                    </thead>
+                                    <tbody>
+                                    </tbody>
+                                </table>
+                            </div>
+                            <!-- Evaluations Pane -->
+                            <div class="tab-pane fade show" id="evaluations-{{ course.year }}" role="tabpanel"
+                                 aria-labelledby="evaluations-tab-{{ course.year }}">
+                                <!-- Content for Evaluations -->
+                                <table class="table table-hover" id="evaluationTable">
+                                    <thead>
+                                    <tr>
+                                        <th>Course name</th>
+                                        <th>Year</th>
+                                    </tr>
+                                    </thead>
+                                    <tbody>
+                                    {% if evaluations is none %}
+                                        <tr>
+                                            <td colspan="3">No evaluations for this year.</td>
```

Comment:
```suggestion
                                            <td colspan="2">No evaluations for this year.</td>
```

since this is a 2-column table